src/sentry/web/frontend/auth_login.py

```diff
@@ -262,7 +263,9 @@ def handle_basic_auth(self, request: Request, **kwargs):
             "register_form": register_form,
             "CAN_REGISTER": can_register,
             "join_request_link": self.get_join_request_link(organization),
+            "show_session_replay_banner": not is_self_hosted(),  # show to everyone except self-hosted
```

Comment:
I'm assuming ST is included in self-hosted?

Reply:
@billyvg good question. I will double check.